api/cmd/api/setup.go

```diff
@@ -165,6 +168,10 @@ func initImageBuilder(cfg *config.Config, vaultClient vault.Client) (webserviceB
 		DockerRegistry:       cfg.ImageBuilderConfig.DockerRegistry,
 		ContextSubPath:       cfg.ImageBuilderConfig.PredictionJobContextSubPath,
 		BuildTimeoutDuration: timeout,
+		KanikoImage:          cfg.ImageBuilderConfig.KanikoImage,
+		MaximumRetry:         cfg.ImageBuilderConfig.MaximumRetry,
+		CpuRequest:           cfg.ImageBuilderConfig.CpuRequest,
+		NodePoolName:         cfg.ImageBuilderConfig.NodePoolName,
```

Comment:
Can we accept yaml instead of node name so that merlin's administrator can have better flexibility? Also make the config optional.

Reply:
The configuration should be compatible with the `NodeSelector` field in kubernetes.

Reply:
At first, I thought we are migrating to a file-based configuration. Thinking out loud here, I've been thinking to move from environment variables-based configuration to file-based configuration since the number of our configurations already grew relatively long. But since this topic is out of scope for this PR, we can revisit it later.

For this PR scope, I agree with Aria to have this NodePoolName compliance with Kubernetes' NodeSelector,